Kudu.Core/SiteExtensions/SiteExtensionManager.cs

```diff
@@ -484,7 +484,7 @@ public SiteExtensionManager(IContinuousJobsManager continuousJobManager, ITrigge
                                 Executable exe = externalCommandFactory.BuildCommandExecutable(installScript,
                                     installationDirectory,
                                     _settings.GetCommandIdleTimeout(), NullLogger.Instance);
-                                exe.ExecuteWithProgressWriter(NullLogger.Instance, _traceFactory.GetTracer(), String.Empty);
+                                exe.ExecuteWithProgressWriter(NullLogger.Instance, _traceFactory.GetTracer(), installationArgs ?? String.Empty);
```

Comment:
Wondering are there unit tests in KuDu that needs to be updated for the new potential error cases?